### PR TITLE
Mock metastore

### LINKED ISSUE #46: Make a mock metastore
The metastore takes a long time to build and stand up.

It would be very convenient if there was an option to have a mock backend for the metastore api that didn't take a long time to setup

### PR DESCRIPTION
**Pull request by jessebmiller**
resolves #46 by introducing a mock metastore.

It's basically the holochain websocket RPC interface naively implemented in pure javascript.

I put some fixes for `make squad-games-web` for new clones too. It removes the dependency on Docker for ganache, and uses npx instead.

### CODE REVIEW

File: packages/metastore/clients/js/index.js [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
+    squad.connection = mockMetastore()
+  } else {
+    squad.connection = new WebSocket(uri)
+  }
   return squad.connection
 }
 
-function mockConnection(mock) {
-  squad.connection = mock
+function mockConnection() {
```
Comment (jessebmiller) [inline packages/metastore/clients/js/index.js]: This is redundant with `webSocketConnection('mock')`

**Comment by EzraWeller** [comment]
Code looks great to me. My only question is: what is our plan for how to use this? Maybe we can update one of the READMEs with some info about when you would want to use this mock connection during the dev process?

**Review by EzraWeller — APPROVED** [review]

File: packages/metastore/clients/js/index.js (hunk 1 above)
Comment (jessebmiller) [reply packages/metastore/clients/js/index.js]: removed

**Comment by jessebmiller** [comment]
Did a little refactoring, and updated CONTRIBUTING.md with instructions.

Going to merge now, and we can revisit if you see areas for improvement, or still have questions the documentation didn't answer.

**Pull request merged**